src/openrct2/peep/Staff.cpp

```diff
@@ -493,8 +491,8 @@ static uint8_t staff_handyman_direction_to_nearest_litter(Peep* peep)
         }
     } while (!(tileElement++)->IsLastForTile());
 
-    nextTile.x = (peep->x & 0xFFE0) + CoordsDirectionDelta[nextDirection].x;
-    nextTile.y = (peep->y & 0xFFE0) + CoordsDirectionDelta[nextDirection].y;
+    nextTile.x = (x & 0xFFE0) + CoordsDirectionDelta[nextDirection].x;
```

Comment:
```suggestion
    nextTile = CoordsXY(x, y).ToTileStart() + CoordsDirectionDelta[nextDirection];
```

If we're touching this line, might as well clean it up.